pxr/usd/usdSkel/validators.cpp

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/usd/sdr/shaderProperty.h"
```

Comment:
Similarly sdr related headers shouldn't be required in usdSkel.

So this one and the sdr/registry.h below.

Reply:
removed